resources/logging/templates/loki/servicerolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRoleBinding
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  subjects:
+  - user: "cluster.local/ns/kyma-system/sa/logging-loki"
```

Comment:
we should also update clusterrbacconfig file with updated service name, file placed in the chart resources/core/charts/istio-rbac/templates/rbac-config.yaml

Reply:
done